phpstan.neon.dist

```diff
@@ -50,3 +52,45 @@ parameters:
         -
             message: '#.*deprecated.*#'
             path: '*/update.php'
+        # ignore query errors in the temporary rex_tests table
+        - '#^Query error.*rex_tests.*#'
+
+services:
+# redaxos setQuery etc. APIs support both, prepared and unprepared statements..
+# therefore we need to register them twice, to also cover cases which don't pass parameters.
+    -
+        class: staabm\PHPStanDba\Rules\SyntaxErrorInPreparedStatementMethodRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            classMethods:
+                - 'rex_sql::setQuery'
+                - 'rex_sql::setDBQuery'
+                - 'rex_sql::getArray'
+                - 'rex_sql::getDBArray'
+
+    -
+        class: staabm\PHPStanDba\Rules\SyntaxErrorInQueryMethodRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            classMethods:
+                - 'rex_sql::setQuery#0'
+                - 'rex_sql::setDBQuery#0'
+                - 'rex_sql::getArray#0'
+                - 'rex_sql::getDBArray#0'
+                - 'rex_sql::prepareQuery#0'
+                - 'rex_sql::getQueryType#0'
```

Comment:
aufgrund von https://github.com/staabm/phpstan-dba/pull/245 kann hier ggf. die doppelte konfiguration in `SyntaxErrorInPreparedStatementMethodRule` und `SyntaxErrorInQueryMethodRule` jetzt wegfallen